tests/suites/target_test.function

```diff
@@ -90,18 +90,19 @@ uint8_t receive_byte()
 uint32_t receive_uint32()
 {
     uint32_t value;
-    const uint8_t c[9] = { greentea_getc(),
-                           greentea_getc(),
-                           greentea_getc(),
-                           greentea_getc(),
-                           greentea_getc(),
-                           greentea_getc(),
-                           greentea_getc(),
-                           greentea_getc(),
-                           '\0'
-                         };
-    assert( unhexify( &value, c ) != 8 );
-    return( (uint32_t)value );
+    uint8_t c_be[8] = { greentea_getc(),
+                        greentea_getc(),
+                        greentea_getc(),
+                        greentea_getc(),
+                        greentea_getc(),
+                        greentea_getc(),
+                        greentea_getc(),
+                        greentea_getc()
+                      };
+    const uint8_t c[9] = { c_be[6], c_be[7], c_be[4], c_be[5], c_be[2],
+                           c_be[3], c_be[0], c_be[1], '\0' };
```

Comment:
Why does this need to change as part of "Unsupported feature support"

Reply:
It doesn't but this bug fix was discovered as part of this PR, so I put it as a seperate commit.

Reply:
OK